### PR TITLE
[-] MO: paypal / Use new hook with PSCleaner module

### PR DESCRIPTION
**Pull request by PrestaEdit**

### CODE REVIEW

File: paypal.php [hunk 1]
```diff
@@ -124,7 +124,7 @@ public function install()
 		if (!parent::install() || !$this->registerHook('payment') || !$this->registerHook('displayPaymentEU') || !$this->registerHook('paymentReturn') ||
 		!$this->registerHook('shoppingCartExtra') || !$this->registerHook('backBeforePayment') || !$this->registerHook('rightColumn') ||
 		!$this->registerHook('cancelProduct') || !$this->registerHook('productFooter') || !$this->registerHook('header') ||
-		!$this->registerHook('adminOrder') || !$this->registerHook('backOfficeHeader'))
+		!$this->registerHook('adminOrder') || !$this->registerHook('backOfficeHeader') || !$this->registerHook('actionPSCleanerGetModulesTables')h)
```
Comment (Quetzacoalt91) [inline paypal.php]: Hi, we have a PHP syntax error here ;)
Can you please remove the `h` ?

**Comment by PrestaEdit** [comment]
Hi @Quetzacoalt91 . Oops, sorry. Done ! :)

**Comment by thoma202** [comment]
Hi, 

sorry for the long wait. 
Could you quickly explain what this is for ? As I'm not aware of this new hook. 

Thanks. 
Thoma

**Comment by gaillafr** [comment]
Hi @thoma202,

PSCleaner is used when you want to erase all of your test data before you put your shop live.
In this case we want to empty the two db tables.

Thx.

**Comment by thoma202** [comment]
Just for information this is going to be triggered when user removes maintenance mode ?

**Comment by gaillafr** [comment]
No, it should not.
Only when the merchant wants to use the PSCleaner module.

**Comment by thoma202** [comment]
Ok, so it's a manual action ?

**Comment by Quetzacoalt91** [comment]
Hi @PrestaEdit,

Thanks for your contribution. Your PR has been merged with the 3.9.0 of the module (#11) and should be available from tomorrow.

Best regards

**Comment by PrestaEdit** [comment]
Hi !

Thanks, :+1:

**Pull request merged**